quarkus-picocli/src/test/java/OpenShiftPicocliIT.java

```diff
@@ -0,0 +1,5 @@
+import io.quarkus.test.scenarios.OpenShiftScenario;
+
+@OpenShiftScenario
+public class OpenShiftPicocliIT extends PicocliIT {
```

Comment:
As far as I understand, we do not plan to run these tests on openshift . Do not forget to remove the file